src/components/NoPeopleMessage/NoPeopleMessage.tsx

```diff
@@ -0,0 +1,5 @@
+export const NoPeopleMessage = () => (
+  <p data-cy="noPeopleMessage">
```

Comment:
It's not really necessary to create a separate component for one string. But it's up to you in this case